stubs/Flask/flask/app.pyi

```diff
@@ -1,22 +1,8 @@
+from contextlib import AbstractContextManager
 from datetime import timedelta
 from logging import Logger
 from types import TracebackType
-from typing import (
-    Any,
-    ByteString,
-    Callable,
-    ContextManager,
-    Dict,
-    Iterable,
-    List,
-    NoReturn,
-    Optional,
-    Text,
-    Tuple,
-    Type,
-    TypeVar,
-    Union,
-)
```

Comment:
`typing.ContextManager` is the only one that's been removed from this list; the diff is only so big because of black.